provisioner/terraform/testdata/chaining-resources/chaining-resources.tfplan.json

```diff
@@ -79,29 +80,29 @@
           "connection_timeout": 120,
           "dir": null,
           "env": null,
+          "login_before_ready": true,
```

Comment:
I think we removed these in #16309 and should not be added back.

Reply:
These are auto-gen'd by terraform, I can't control if they're re-added or not. Were they removed manually before?

Reply:
The PR you linked is slightly confusing. I don't see anywhere that it's updating the provider versions

Reply:
@defelmnq can you double-check what's going on here?

Reply:
Also, we still have `v1.0.4` of the provider in `go.mod`. So this is confusing me, too. I was of the impression we updated the provider to v2. 
https://github.com/coder/coder/blob/1ce4dfe0583cd74356d7dde8215f95e20a527b02/go.mod#L97

Reply:
I think it might be related to these log lines when I run generate.sh:

```
╷
│ Error: Unsupported block type
│
│   on multiple-agents-multiple-monitors.tf line 13, in resource "coder_agent" "dev1":
│   13:   resources_monitoring {
│
│ Blocks of type "resources_monitoring" are not expected here.
╵
╷
│ Error: Unsupported block type
│
│   on multiple-agents-multiple-monitors.tf line 24, in resource "coder_agent" "dev2":
│   24:   resources_monitoring {
│
│ Blocks of type "resources_monitoring" are not expected here.
```

Reply:
`resources_monitoring` was added into `main` and has not been tagged yet.